bears/python/PEP8NotebookBear.py

```diff
@@ -0,0 +1,116 @@
+import autopep8
+import nbformat
+
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+# Comments regarind Jupyter Notebooks:
+# The `nbformat` module contains the reference implementation of the Jupyter
+# Notebook format, and Python APIs for working with notebooks.
+# On the file level, a notebook is a JSON file, i.e. dictionary with a few
+# keys.
+# The functions in `nbformat` work with `NotebookNode` objects, which are like
+# dictionaries, but allow attribute access. The structure of these objects
+# matches the notebook format specification.
+
+
+def read_notebook_node_from_file(file):
+    """
+    Reads a notebook from a string array and returns the NotebookNode
+    object.
+
+    :param file: The notebook file contents as string array.
+    :return:     The notebook as NotebookNode.
+    """
+    return nbformat.reads(''.join(file), nbformat.NO_CONVERT)
+
+
+def write_notebook_node_to_file(notebook_node):
```

Comment:
I will give `notebook_node_to_list` and `notebook_node_from_list` a try. Should not sound fancier than it is...